src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -27,6 +27,8 @@ partial class WorkspaceViewModel
 
         private StateMachine stateMachine = null;
         private ConnectorViewModel activeConnector = null;
+        private ConnectorViewModel[] activeConnectors = null;
```

Comment:
Here again, activeConnectors could be used for activeConnector also. And if activeConnectors size is more than one then it's a case of multipleConnection, do we need a separate bool value to store this data?